### PR TITLE
fix(styles,documentation): Improved rendering of the home page for HCM

### PR DESCRIPTION
**Pull request by imagoiq**

### CODE REVIEW

File: packages/styles/src/mixins/_notification.scss [hunk 1]
```diff
@@ -72,7 +72,6 @@
 
   @include utilities-mx.high-contrast-mode {
     filter: none !important;
-    background-color: transparent !important;
     border: 2px solid WindowText;
```
Comment (oliverschuerch) [inline packages/styles/src/mixins/_notification.scss:75]:
```suggestion
    border: 2px solid CanvasText;
```
https://wiki.mozilla.org/Accessibility/Design_Guide ->
"Components not in the above categories should use CanvasText / Canvas (older implementations may use WindowText / Window)"

**Review by oliverschuerch — CHANGES_REQUESTED** [review]
The SVG package icons are not working in dark-mode (not HCM related).
Do you have any idea how to fix this?
image

File: packages/styles/src/mixins/_notification.scss (hunk 1 above)
Comment (imagoiq) [reply packages/styles/src/mixins/_notification.scss]: Good catch!

**Comment by imagoiq** [comment]
> The SVG package icons are not working in dark-mode (not HCM related). Do you have any idea how to fix this? image

@oliverschuerch I can't reproduce it. Do you use the rendering panel on Chrome or something else?

**Review by oliverschuerch — APPROVED** [review]

**Pull request merged**